src/components/change-password/tests/change-password-controller.test.ts

```diff
@@ -55,5 +67,30 @@ describe("change password controller", () => {
         PATH_DATA.PASSWORD_UPDATED_CONFIRMATION.url
       );
     });
+    it("should render bad request when password are same ", async () => {
+      const fakeService: ChangePasswordServiceInterface = {
+        updatePassword: sandbox.fake.returns({
+          success: false,
+          code: ERROR_CODES.NEW_PASSWORD_SAME_AS_EXISTING,
+        }),
+      };
+      req.session.user.tokens = { accessToken: "token" };
+      await changePasswordPost(fakeService)(req as Request, res as Response);
+
+      expect(res.status).to.have.calledWith(HTTP_STATUS_CODES.BAD_REQUEST);
+    });
+
+    it("should render bad request when password is common ", async () => {
+      const fakeService: ChangePasswordServiceInterface = {
+        updatePassword: sandbox.fake.returns({
+          success: false,
+          code: ERROR_CODES.PASSWORD_IS_COMMON,
+        }),
+      };
+      req.session.user.tokens = { accessToken: "token" };
+      await changePasswordPost(fakeService)(req as Request, res as Response);
+
+      expect(res.status).to.have.calledWith(HTTP_STATUS_CODES.BAD_REQUEST);
```

Comment:
Should the test also confirm the error content that will be displayed on the front end?